common/lib/capa/capa/templates/choicetext.html

```diff
@@ -10,7 +11,7 @@
 <form class="choicetextgroup capa_inputtype" id="inputtype_${id}">
     <div class="script_placeholder" data-src="${STATIC_URL}js/capa/choicetextinput.js"/>
 
-    <fieldset aria-label="${response_data['label']}">
+    <fieldset aria-label="${remove_markup(response_data['label'])}">
```

Comment:
Here I think it would be much preferred to have `${remove_markup(response_data['label_html'])}` if possible, because label does not seem like something that would contain HTML.